js/src/carousel.js

```diff
@@ -231,6 +233,23 @@ const Carousel = (($) => {
       return config
     }
 
+    _getTransitionDuration() {
+      // Get transition-duration of first element in the carousel
+      let transitionDuration = $(this._element).find(Selector.ITEM).css('transition-duration')
+
+      // Return 0 carousel item is not found
+      if (!transitionDuration) {
+        return 0
+      }
+
+      // If multiple durations are defined, take the first
+      transitionDuration = transitionDuration.split(',')[0]
+
+      // Multiply by 1000 if transition-duration is defined in seconds
+      return transitionDuration.indexOf('ms') > -1 ?
```

Comment:
If you use `includes` here, does babel take care of it or do we need a polyfill?

Reply:
Or even `endsWith`?

Reply:
@XhmikosR, just tested this, `includes` and `endsWith` are not polyfilled by babel.

Reply:
Thanks for testing. Not sure if we need to enable something else in our babel config, but it gets out of this PRs scope anyway.